### PR TITLE
Small fixes and suggestions

### PR DESCRIPTION
**Pull request by scordio**
I needed to bootstrap my knowledge about JUnit 5 custom extensions, and I stumbled upon [this great article](https://nipafx.dev/junit-5-extension-model/). Thanks for putting it together!

During the reading, I noticed a couple of things I'd change so I'm raising this PR with my suggestions.

### CODE REVIEW

**Comment by nipafx** [comment]
Hey @scordio, thank you for the PR 🤩 and excuse the late reply. I seem to have missed the notification mail (no idea how that happened!) but I really appreciate your contribution. I'll take a look at it ASAP.

File: content/articles/2018-08-05-junit-5-extension-model.md [hunk 1]
```diff
@@ -296,12 +296,12 @@ If you have an extension that you think needs to be registered with all tests in
...
-While your at it, consider requiring explicit activation for your extension with your own parameter (you can query it with the store's `getConfigurationParameter` method).
+While you're at it, consider requiring explicit activation for your extension with your own parameter (you can query it with the store's `getConfigurationParameter` method).
 This way you can use your extension JAR without all global extensions being registered all the time.
 
 ### Custom Annotations
 
-The JUnit Jupiter API is driven by annotations and the engine does a little extra work when it checks for their presence: It not only looks for annotations on classes, methods and parameters but also *on other annotations*.
+The JUnit Jupiter API is driven by annotations, and the engine does a little extra work when it checks for their presence: it looks for annotations not only on classes, methods and parameters but also *on other annotations*.
```
Comment (nipafx) [inline content/articles/2018-08-05-junit-5-extension-model.md:304]: Oh, I have to remember this one. I think I make this mistake quite frequently (when I use that phrase).

**Comment by nipafx** [comment]
Thank you very much Stefano, I really appreciate you taking the time to fix these mistakes!

**Comment by scordio** [comment]
No worries, happy that it was helpful 🙂

**Pull request merged**